overlay/mobile-nixos/kernel/builder.nix

```diff
@@ -86,6 +87,9 @@ in
 # Mark the kernel as compressed, assumes .gz
 , isCompressed ? "gz"
 
+# Enable build of dtbo.img
+, dtboImg ? false
```

Comment:
For now `true` is the only *expected* value...

But I'm leaving the door open to a *pattern* to be used instead so that only *some* dtbo files are included in the build. That pattern would be given to the `find` command.